### PR TITLE
Update runtime to 5.15-23.08

### PR DESCRIPTION
**Pull request by liushuyu**
This pull request updates the KDE runtime to version 5.15-23.08.

### CODE REVIEW

**Comment by liamwhite** [comment]
Be sure to also update the readme

**Comment by liushuyu** [comment]
> Be sure to also update the readme

Thank you for the reminder! Will do.

**Comment by liushuyu** [comment]
bot, build org.yuzu_emu.yuzu

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Obtaining a debugger trace is a bit complicated, so please bear with us:
 
 #### Obtaining a debugger trace as the crash happens
 
-1. Install debuggers in Flatpak: `flatpak install org.kde.Sdk//5.15-22.08`
+1. Install debuggers in Flatpak: `flatpak install org.kde.Sdk//5.15-23.08`
```
Comment (liamwhite) [inline README.md:34]: Missed the one above

Comment (liushuyu) [reply README.md:34]: Done

**Comment by liushuyu** [comment]
The bot is stuck updating the statuses again. Retrying.

**Comment by liushuyu** [comment]
bot, build org.yuzu_emu.yuzu

**Pull request merged**